sdk/containerregistry/container-registry/swagger/README.md

```diff
@@ -23,3 +23,50 @@ use-extension:
   "@autorest/typescript": "6.0.0-beta.4"
 package-version: 1.0.0-beta.6
 ```
+
+## Customizations for Track 2 Generator
+
+See the [AutoRest samples](https://github.com/Azure/autorest/tree/master/Samples/3b-custom-transformations)
+for more about how we're customizing things.
+
+### Remove response for "ContainerRegistry_DeleteRepository" operation
```

Comment:
Could we add some brief remarks about why we're doing these various swagger changes? I feel like it would be useful for future maintainers

Reply:
Added some comments